docs/create/developer-tools/arch3js.md

```diff
@@ -0,0 +1,79 @@
+---
+sidebar_position: 3
+title: arch3.js
+---
+
+# arch3.js 
+
+[`arch3.js`](https://www.npmjs.com/package/@archwayhq/arch3-core) is a JavaScript library that enables the interaction between front-end web applications and the Archway Protocol. It is a replacement using the [CosmJS](https://github.com/cosmos/cosmjs) clients. 
+
+
+## Installation 
+### NPM  
+```bash 
+npm install --save @archwayhq/arch3.js
+```
+
+### Yarn 
+```bash 
+yarn add @archwayhq/arch3.js
+```
+
+## Sample Usage 
+
+### Querying 
+Querying the blockchain is a way to understand the events and statuses that happen inside a smart contract. Below is an example using the increment contract template to Constantine. 
+
+```js 
+import { ArchwayClient } from '@archwayhq/arch3.js';
+
+const client = await ArchwayClient.connect('https://rpc.constantine-1.archway.tech');
+
+const contractAddress = 'archway14v952t75xgnufzlrft52ekltt8nsu9gxqh4xz55qfm6wqslc0spqspc5lm';
+const msg = {
+  get_count: {},
+};
+const { count } = await client.queryContractSmart(
+  contractAddress,
+  msg
+);
+```
```

Comment:
when trying it out creating a `main.js` file , i had to add the `"type":  "module"` in the `package.json`. (as i was getting the `SyntaxError: Cannot use import statement outside a module`)

i then ran again the `node main.js` command and got into the:
`Error: Query failed with (18): contract: not found: invalid request`

Is the contract up on constantine?

I changed the contract address to another i instanciated, and it correctly worked (my contract address: `archway1dd8xfz4msg7aufxcc4sjjzusg4rsp98kza2z7zjprhgwvmpdzncq6jrg99`)